File: site/script/signup.php
 <?php
$servername = "localhost";
$username = "root";
$password = "";
$dname="site";
$conn=new mysqli($servername,$username,$password,$dname);
echo $_POST['user'];
echo $_POST['password'];
echo $_POST['name'];
echo $_POST['mail'];

if (!isset($_POST['admin']))
$aux=0;
else $aux=$_POST['admin'];
$password=hash('sha3-512',$_POST['password']);
echo $password."<br>".strlen($password);
$sql="INSERT INTO user (NAME,USERNAME,PASSWORD,MAIL,ADMIN) VALUES ('".$_POST['name']."','".$_POST['user']."','".$password."','".$_POST['mail']."',".$aux.")";
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);}
if($conn->query($sql)==TRUE)
echo "User added";
else {echo "User <b>not</b> added<br>";
     echo "Error: " . $sql . "<br>" . $conn->error."<br>";}
$conn->close();
echo "Connected successfully";
?> 

File: site/scriptaux/signup.py
#!/Python27/python
import cgi, cgitb
import MySQLdb
form = cgi.FieldStorage() 
name = form.getvalue('name')
user  = form.getvalue('user')
password  = form['password'].value
mail  = form.getvalue('mail')
db = MySQLdb.connect("localhost","root","","site")
cursor = db.cursor()
sql="INSERT INTO user (NAME,USERNAME,PASSWORD,MAIL,ADMIN) VALUES (%s,%s,%s,%s,0)" % (name,user,password,mail)
print "Content-type:text/html\r\n\r\n"
print "<html><body>",sql
try:
 cursor.execute(sql)
 db.commit()
except:
 print "NO DBASE <br>"
db.close()
print "</body></html>"
